packages/mui-material/src/Chip/Chip.d.ts

```diff
@@ -72,10 +72,20 @@ export interface ChipTypeMap<P = {}, D extends React.ElementType = 'div'> {
      * @default 'medium'
      */
     size?: OverridableStringUnion<'small' | 'medium', ChipPropsSizeOverrides>;
+    /**
+     * If `true`, allows the disabled chip to escape focus.
+     * If `false`, allows the disabled chip to receive focus.
+     * @default false
+     */
+    skipFocusWhenDisabled?: boolean;
```

Comment:
Should we add a `TODO` for v6 to rename the prop to `focusableWhenDisabled`, where we will turn off focus by default when the chip is disabled?
```suggestion
    /**
     * If `true`, allows the disabled chip to escape focus.
     * If `false`, allows the disabled chip to receive focus.
     * @default false
     * TODO v6: Rename to `focusableWhenDisabled`.
     */
    skipFocusWhenDisabled?: boolean;
```

Reply:
Yes, that's a good idea, but not in JSDoc. Let's add a separate comment with just the TODO